CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ All notable changes to this project will be documented in this file.
 - `UVData.order_pols` method in favor of `UVData.reorder_pols`.
 
 ### Fixed
+- Changed the way interpolation splines are saved in UVBeam to fix errors related to polarization selections.
```

Comment:
This should go under 'unrealeased' rather than under 1.3.8 which was released Wednesday.

Reply:
Whoops... should be fixed now.